VAMobile/src/screens/HealthScreen/Pharmacy/RefillScreens/RefillRequestSummary.tsx

```diff
@@ -124,11 +124,11 @@ function RefillRequestSummary({ navigation, route }: RefillRequestSummaryProps)
 
   const getRequestSummaryItem = () => {
     return refillRequestSummaryItems.map((request, index) => {
-      const vaIconProps: VAIconProps = {
-        name: request.submitted ? 'CircleCheckMark' : 'Remove',
+      const iconProps: IconProps = {
+        name: request.submitted ? 'Check' : 'Close',
         width: 20,
         height: 20,
-        fill: request.submitted ? theme.colors.icon.success : theme.colors.icon.error,
+        fill: theme.colors.icon.pagination,
```

Comment:
Any potential confusion by using the pagination color here?

Reply:
Honestly there shouldn't be. It was only used cause it is white for dark and light mode and I don't think there is another color for it. I know some of our hard coded ones are #FFF to represent the same thing. So I don't think we would have any issues here.